website/docs/language/stacks/reference/tfdeploy.mdx

```diff
@@ -312,3 +312,111 @@ You can access specific environment variables by key from the `store.varset.avai
 ## `locals` block configuration
 
 A local value assigns a name to an expression, so you can use the name multiple times within your Stack configuration instead of repeating the expression. The `locals` block works exactly as it does in traditional Terraform configurations. Learn more about [the `locals` block](/terraform/language/values/locals).
+
+## `publish_output` block configuration
+
+-> **Note**: To use the `publish_output` block requires at least Terraform version `terraform_1.10.0-alpha20241009` or higher. We recommend downloading the [latest version of Terraform](https://releases.hashicorp.com/terraform/) to use the most up-to-date functionality.
+
+To export information from one Stack to another, declare the `publish_output` block in your deployment configuration to export data that another Stack can intake.
```

Comment:
```suggestion
Specifies a value to export from your Stack. Refer to [Pass data from one Stack to another](//terraform/language/stacks/deploy/link-stacks) for additional information. 
```
The reference should just describe the behavior.

Reply:
I'm going to split the difference a bit to give folks some context here:

```suggestion
Specifies a value to export from your current Stack, which other Stacks in the same project can consume. Declare one `publish_output` block for each value to export your Stack. 
```